instance-scheduler/main_test.go

```diff
@@ -1,29 +1,17 @@
 package main
 
 import (
-	"fmt"
 	"github.com/aws/aws-lambda-go/events"
-	"net/http"
-	"net/http/httptest"
-	"os"
 	"testing"
 )
 
 func TestHandler(t *testing.T) {
-	os.Setenv("INSTANCE_SCHEDULING_ACTION", "Test")
```

Comment:
I think it's preferable to explicitly set the action to "Test" to make sure the lambda will not actually stop or start instances during the test.

Reply:
I actually added it back with my next commit (inside the test subtest), but have never pushed it. It's there now.

Reply:
Thank you